packages/apps/explorer/.eslintrc.js

```diff
@@ -3,6 +3,14 @@
 require('@rushstack/eslint-config/patch/modern-module-resolution');
 
 module.exports = {
-  extends: ['@kadena/eslint-config/profile/lib'],
+  extends: [
+    '@kadena/eslint-config/profile/lib',
+    'eslint-config-airbnb-typescript',
+    '@next/eslint-plugin-next',
+    'eslint-config-prettier',
```

Comment:
Not sure if this is necessary. We should have an automatic prettier run on commit, therefore there's no need to have also a linter. Unless this is for the other plugins? https://github.com/kadena-community/kadena.js/blob/master/common/git-hooks/pre-commit